postprocess/postprocess.go

```diff
@@ -0,0 +1,228 @@
+// Package postprocess contains functionality to postprocess pointcloud maps
+package postprocess
+
+import (
+	"bytes"
+	"errors"
+	"image/color"
+
+	"github.com/golang/geo/r3"
+	"go.viam.com/rdk/pointcloud"
+)
+
+// Instruction describes the action of the postprocess step.
+type Instruction int
+
+const (
+	// Add is the instruction for adding points.
+	Add Instruction = iota
+	// Remove is the instruction for removing points.
+	Remove = iota
+)
+
+const (
+	fullConfidence = 100
```

Comment:
Right now i set all the added points to have 100% confidence. I did this because I figured if a user added it that number made sense. I could see an argument for having this be configurable but opted for this to simplify the `DoCommand`. What to others think?

Reply:
I think using full confidence makes sense. Cannot think of a use case where we would want it to be different